style.css

```diff
@@ -462,12 +498,396 @@ textarea {
+    background-position: right -1em top -250px;
+  }
+
+  .title {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 81px;
+    color: #f55800;
+  }
+
+  .subtitle {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .illustration {
+    padding-right: 30px;
+  }
+
+  .description {
+    display: grid;
+    grid-template-columns: 500px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .social-media ul {
+    display: inline;
+    position: absolute;
+    left: 2%;
+    top: 30%;
+    padding-right: 60px;
+  }
+
+  .card-single {
+    display: flex;
+  }
+
+  .head {
+    border-bottom: none;
+  }
+
+  .main-butt {
+    padding-left: 20px;
+  }
+
+  .about-dec {
+    display: flex;
+    padding-left: 100px;
+    padding-bottom: 60px;
+  }
+
+  .border {
+    display: none;
+  }
+
+  .titl {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 81px;
+    padding-left: 30px;
+    text-align: start;
+  }
+
+  .about-me {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    margin-left: 25px;
+  }
+
+  .single-work img {
+    width: 600px;
+  }
+
+  .all-projects {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 20px;
+    justify-content: center;
+    padding-top: 30px;
+    padding-left: 200px;
+    align-items: center;
+  }
+
+  .about-ill {
+    padding-top: 120px;
+    align-items: center;
+  }
+
+  .contact {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .contact-desc h3 {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+    color: #262626;
+  }
+
+  .contact-form {
+    padding-top: 60px;
+  }
+
+  input {
+    outline: none;
+    font-size: 15px;
+    line-height: 24px;
+    padding-top: 50px;
+    font-style: normal;
+    font-weight: normal;
+    color: #979493;
+    padding-bottom: 15px;
+    padding-left: 25px;
+  }
+
+  input::placeholder {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    color: #979493;
+  }
+
+  textarea {
+    line-height: 24px;
+    background: #fbf8f7;
+    font-size: 15px;
+    margin-right: 24px;
+    color: #3c3a39;
+    font-style: normal;
+    font-weight: normal;
+    padding-top: 20px;
+    margin-bottom: 30px;
+    margin-left: 25px;
+  }
+
+  .mess textarea::placeholder {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 24px;
+    color: #3c3a39;
+  }
+
+  .btn {
+    background-color: #ff6b00;
+    width: 20%;
+    padding: 12px;
+    height: 48px;
+    margin-bottom: 100px;
+    margin-left: 30px;
+  }
+
+  .lang-part {
+    display: flex;
+  }
+
+  .skills ul {
+    list-style: none;
+  }
```

Comment:
- [ ] Kindly stick to the Figma design as much as you can, the left and the right margins of the contact section are not the same as the margins that are provided in the Figma design. Kindly fix that.
- [ ] The width of the `textarea` of your form is not as other inputs. Please work on it.
- [ ] Please align the line that is under the social icons to the center of the page.
- Your design
<img width="424" alt="image" src="https://user-images.githubusercontent.com/77190203/154688515-a7b1e3dd-384f-499b-a9b5-f72f1c0483b6.png">

- Figma design:
<img width="394" alt="image" src="https://user-images.githubusercontent.com/77190203/154688756-81a30025-d00c-4d24-9a70-a8b3d4d16243.png">

Reply:
I decided to use another template